src/helm/benchmark/static/schema_vhelm.yaml

```diff
@@ -474,6 +462,102 @@ run_groups:
       when: "2023"
       language: Swahili
 
+  - name: mm_star_perception
```

Comment:
Just to make sure, we want to split both MM-Star and BLINK into 3 different evaluaiton aspects (perception, reasoning, and knowledge), right?

Reply:
yes that's correct